### PR TITLE
fix: Increase the Tars protocol number in the Server section

### PR DESCRIPTION
**Pull request by incoffeemonster**

### CODE REVIEW

**Comment by sharang** [comment]
pr title 中，冒号后面加个空格

**Comment by incoffeemonster** [comment]
done

File: server/querier/db_descriptions/clickhouse/tag/enum/l7_protocol [hunk 1]
```diff
@@ -7,6 +7,7 @@
 43      , SOFARPC         ,
 44      , FastCGI         ,
 45      , bRPC            ,
+45      , Tars            ,
```
Comment (sharang) [inline server/querier/db_descriptions/clickhouse/tag/enum/l7_protocol:10]: 数字不对

Comment (incoffeemonster) [reply server/querier/db_descriptions/clickhouse/tag/enum/l7_protocol]: done

**Review by rvql — APPROVED** [review]

**Pull request merged**